assets/src/dashboard/components/contentGutter/index.js

```diff
@@ -23,11 +23,11 @@ export const StandardViewContentGutter = styled.div(
   ({ theme }) => `
     margin: 0 ${theme.DEPRECATED_THEME.standardViewContentGutter.desktop}px;
```

Comment:
There are still old styles here that use the old design system. Do we want to update this now or later?

For example, this value is `52` but we don't have a constant for that in the design system. We'd just have to do 
```suggestion
    margin: 0 52px;
```

Reply:
i think swapping to 52 makes sense here, it's contained and easy to figure out.  I vote update now.